chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -103,6 +103,10 @@ async def zero_calls_get_coin_records(coin_ids: Collection[bytes32]) -> list[Coi
     return []
 
 
+async def zero_calls_get_unspent_lineage_info_for_puzzle_hash(_puzzle_hash: bytes32) -> Optional[UnspentLineageInfo]:
+    assert False  # pragma no cover
```

Comment:
if the test expects this to never be called, I think you should put an `assert False` here, and maybe with a comment explaining why we don't expect it to be called

Reply:
This function already does `assert False` and it's used in `instantiate_mempool_manager` to conveniently omit it when a test calls `instantiate_mempool_manager` with no need to call that function.

Reply:
oh, heh. I must have been blinded by "pragma no cover"